_data/conferences.yml

```
@@ -2015,6 +2015,13 @@
   twitter: euruko
   cfp_phrase: CFP closes
   cfp_date: 2021-04-18
+
+- name: Saint P Rubyconf 2021
```

The site style is to remove year from the name of the events since it's in the start/end dates. ❤️ 

```suggestion
- name: Saint P Rubyconf
```